### PR TITLE
Rolling PR for flight-job 1.2.X

### PR DESCRIPTION
**Pull request by WilliamMcCumstie**
RONSEAL

### CODE REVIEW

File: builders/flight-job/opt/flight/libexec/job/flight-slurm/monitor.sh [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+#
+#  https://opensource.org/licenses/EPL-2.0
+#
+# For more information on OpenFlight Omnibus Builder, please visit:
+# https://github.com/openflighthpc/openflight-omnibus-builder
+#===============================================================================
+
+# Ensure that flight-slurm binaries are first on the PATH and thus override
+# any other installs of slurm
+PATH=/opt/flight/opt/slurm/bin:$PATH
```
Comment (benarmston) [inline builders/flight-job/opt/flight/libexec/job/flight-slurm/monitor.sh:30]: How did you determine that this was the correct order?  The file `/opt/flight/etc/profile.d/90-slurm.sh` does it the other way around.  We ought to be consistent between the two.

File: builders/flight-job/opt/flight/libexec/job/flight-slurm/submit.sh [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+#
+#  https://opensource.org/licenses/EPL-2.0
+#
+# For more information on OpenFlight Omnibus Builder, please visit:
+# https://github.com/openflighthpc/openflight-omnibus-builder
+#===============================================================================
+
+# Ensure that flight-slurm binaries are first on the PATH and thus override
+# any other installs of slurm
+PATH=/opt/flight/opt/slurm/bin:$PATH
```
Comment (benarmston) [inline builders/flight-job/opt/flight/libexec/job/flight-slurm/submit.sh:30]: Ditto.

**Review by benarmston — CHANGES_REQUESTED** [review]
This looks close to being ready to go.  I've made some changes to the `howto` and added a couple of items for you to look at.  There is also the script templates that need some minor work as mentioned in real life.

File: builders/flight-job/opt/flight/libexec/job/flight-slurm/submit.sh (hunk 2 above)
Comment (WilliamMcCumstie) [reply builders/flight-job/opt/flight/libexec/job/flight-slurm/submit.sh]: https://github.com/openflighthpc/openflight-omnibus-builder/pull/89/commits/7b9ba14e889245dcc5d63b6b04ef86133935e4a7

File: builders/flight-job/opt/flight/libexec/job/flight-slurm/monitor.sh (hunk 1 above)
Comment (WilliamMcCumstie) [reply builders/flight-job/opt/flight/libexec/job/flight-slurm/monitor.sh]: https://github.com/openflighthpc/openflight-omnibus-builder/pull/89/commits/7b9ba14e889245dcc5d63b6b04ef86133935e4a7

**Pull request merged**